runtime/src/append_vec.rs

```diff
@@ -70,6 +71,17 @@ impl<'a> StoredAccount<'a> {
             hash: *self.hash,
         }
     }
+
+    fn sanitize(&self) -> bool {
+        // Sanitize executable
+        // Use extra references to avoid value silently cramped to 1 (=true) and 0 (=false)
```

Comment:
I think you mean "clamped"?
```suggestion
        // Use extra references to avoid value silently clamped to 1 (=true) and 0 (=false)
```

Reply:
yeah... e0aca2aa957b27ce670469f5bcfe8649791ca3f6